backend/src/url_shortener/services/shortening/factory.py

```
@@ -28,57 +34,75 @@ class BaseCreator(ABC):
 	def __init__(self, max_length: int) -> None:
 		super().__init__()
 		self.max_length = max_length
+
 	def factory_method(self) -> Shortener:
 		return CounterEncodedString(self.max_length)
 
+
 """
 USAGE (client):
 	RandomStringCreator(max_length=8).shorten_url(original_url="http://www.my_url.com", 
 								    minutes_until_expiration=100)
```

Ruff (0.14.5)

Indentation contains mixed spaces and tabs

(E101)